subprojects/gradle/impact/src/main/kotlin/com/avito/impact/plugin/ImpactReportTask.kt

```diff
@@ -0,0 +1,31 @@
+package com.avito.impact.plugin
+
+import com.avito.android.sentry.environmentInfo
+import com.avito.android.stats.statsd
+import com.avito.impact.ModifiedProjectsFinder
+import com.avito.impact.configuration.internalModule
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+abstract class ImpactReportTask : DefaultTask() {
+
+    @TaskAction
+    fun doWork() {
+        val modifiedProjectsFinder = ModifiedProjectsFinder.from(project)
+
+        val statsd = project.statsd.get()
+        val environmentInfo = project.environmentInfo().get()
+
+        val impactMetricsSender = ImpactMetricsSender(statsd, environmentInfo)
```

Comment:
In future I want to see a Dependency Inversertion for that `dependency`
It will help to debug

Reply:
fixed, thanks